activesupport/lib/active_support/core_ext/hash/except.rb

```diff
@@ -1,8 +1,14 @@
 class Hash
-  # Returns a hash that includes everything but the given keys.
-  #   hash = { a: true, b: false, c: nil}
-  #   hash.except(:c) # => { a: true, b: false}
-  #   hash # => { a: true, b: false, c: nil}
+  # Returns a hash that includes everything except given keys.
+  #   hash = { a: true, b: false, c: nil }
+  #   hash.except(:c) # => {a: true, b: false}
```

Comment:
As far as I can see from the source of the Rails there is no convention about showing return hash message for ruby but most of time there is no spaces. By the way these are totaly wrong but why? Because ruby will return these hashes like;
`{:a=>1}` not like `{a: 1}` and as you can see there is no spaces before and after brackets. I'm changing these like `{:key=>:value}`.
Are you ok with this @kaspth ?